src/main/scala/io/iohk/ethereum/blockchain/sync/SyncController.scala

```diff
@@ -120,6 +124,8 @@ object SyncController {
   def props(
       appStateStorage: AppStateStorage,
       blockchain: Blockchain,
+      evmCodeStorage: EvmCodeStorage,
+      nodeStorage: NodeStorage,
```

Comment:
I wonder at what point we'll create a case class `MantisEnv` or something that has all these instances in it.

Reply:
We already have something similar `BlockchainStorages` which is extended by `Storages` in the trait `StoragesComponent`, and with a concrete implementation in `DefaultStorages`. 

It is a big bag with every storage in it so we will probably want to separate them to have storages that work together in the same container, and identify storages that are a "view" of other storages.

Reply:
👍